### ISSUE TITLE
at-rule-name-case

### ISSUE THREAD

**Issue by jeddy3** [stylelint/stylelint#1030]
After a little saunter through the specs, there's a few other `*-case` rules pop-up. Hopefully our naming conventions hold true.

```css
/* at-rule-name-case: "upper" */
@MEDIA (min-width: 10px) {}
```

- Name: `at-rule-name-case`
- Primary options: `"upper"|"lower"`
- Secondary options: none
- Message: `Expected "${actual}" to be "${expected}" (at-rule-name-case)`.

**Comment by jeddy3** [comment]
Done in https://github.com/stylelint/stylelint/pull/1072
